draft-thoji-scone-trone-protocol.md

```diff
@@ -310,6 +310,104 @@ This transport parameter is valid for QUIC versions 1 {{QUIC}} and 2
 transport parameters, and frame types registries established in {{Sections 22.2,
 22.3, and 22.4 of QUIC}}.
 
+# TRONE Indication {#trone-indication}
+
+QUIC endpoints can signal potential support for TRONE before the completion of
+the QUIC handshake by attaching a TRONE indication packet after a QUIC Initial
+packet in the first UDP datagram. The TRONE indication provides an
```

Comment:
```suggestion
the QUIC handshake by attaching a TRONE indication packet after the last QUIC long header
packet in the first UDP datagram. The TRONE indication provides an
```

Trone is designed to be QUIC-version-neutral, so it is better to avoid referring to concepts specific to QUIC v1. Also, even in v1, the first datagram that an endpoint sends might contain an Initial and a Handshake (consider servers).